specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json

```diff
@@ -8992,12 +9046,57 @@
       },
       "description": "Describes the properties of a Virtual Machine Scale Set Extension."
     },
+    "VirtualMachineScaleSetExtensionUpdateProperties": {
+      "properties": {
+        "forceUpdateTag": {
+          "type": "string",
+          "description": "If a value is provided and is different from the previous value, the extension handler will be forced to update even if the extension configuration has not changed."
+        },
+        "publisher": {
+          "type": "string",
+          "description": "The name of the extension handler publisher."
+        },
+        "type": {
```

Comment:
Is this updatable?

Reply:
As discussed offline... Not actually, but for few of the scenarios (updating version), it is required to pass in the same type & publisher even during the Patch scenario.
As discussed, using the same VirtualMachineScaleSetExtensionProperties [similar to all the other resources.. VirtualMachine, AvailabilitySet, Image...]